control/control.go

```diff
@@ -61,6 +61,23 @@ func Get(location string, headers []string) (io.ReadCloser, error) {
 			}
 			return res.Body, nil
 		}
+
+		if u.Scheme == "ssm" {
+			Logger.Println("[INFO] Fetching control from SSM location", location)
+			svc, err := NewSSM()
```

Comment:
So, this doesn't pass any AWS keys here, but assuming it works with an instance role. Do you know if it will just work if you have env variables set (AWS_SECRET, etc)?

Reply:
It seems like it will work:

https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html

I haven't tested it yet though.